docs/feature_rgb_matrix.md

```diff
@@ -306,6 +306,12 @@ To declare new effects, create a new `rgb_matrix_user/kb.inc` that looks somethi
 `rgb_matrix_user.inc` should go in the root of the keymap directory.
 `rgb_matrix_kb.inc` should go in the root of the keyboard directory.
 
+To use custom effects in your code, simply prepend `RGB_MATRIX_CUSTOM_` to the effect name specified in `RGB_MATRIX_EFFECT()`. For example, an effect declared as `RGB_MATRIX_EFFECT(my_cool_effect)` would be referenced with:
+
+\`\`\`c
+rgb_matrix_mode(RGB_MATRIX_CUSTOM_my_cool_effect);
+\`\`\`
```

Comment:
Now the backslashes need to be removed.

Reply:
But now I don't have the rights to change it

Reply:
You should be able to change it on your repo.

Reply:
See zvecr's suggestion below.

Reply:
I now saw the 'commit suggestion' option and I did so.